src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -199,13 +212,13 @@ void func_80A74398(Actor* thisx, PlayState* play) {
 
     thisx->colChkInfo.damageTable = &sDamageTable;
     thisx->colChkInfo.mass = MASS_HEAVY;
-    this->unk_2FC = 0;
+    this->isBreakingProp = false;
     thisx->colChkInfo.health = 30;
     thisx->gravity = -1.0f;
-    this->switchFlags = (thisx->params >> 8) & 0xFF;
+    this->switchFlag = IK_GET_SWITCH_FLAG(thisx);
     thisx->params &= 0xFF;
```

Comment:
```suggestion
    thisx->params = IK_GET_ARMOR_TYPE(thisx);
```

Reply:
this one feels a little weird since its more a masking of the existing value, not getting it/extracting it to a new variable. but it still matches so i guess im fine with it?

Reply:
I figured it'd be best for consistency with the `params ==` checks everywhere